src/Data/NormalKey.purs

```diff
@@ -101,6 +97,8 @@ instance ordNormalKey :: Ord NormalKey where
   compare (NormalKey "Shift") _ = LT
   compare _ (NormalKey "Shift") = GT
   compare (NormalKey "Enter") (NormalKey "Enter") = EQ
+  compare _ (NormalKey "Enter") = GT
+  compare (NormalKey "Enter") _ = LT
   compare (NormalKey x) (NormalKey y) = compare x y
```

Comment:
Oh actually I don't think what I wrote does what I wanted. I'd like the keys matched by `(NormalKey s)` to be ordered by their string value and for Enter to come after all other keys.